### ISSUE TITLE
Fibaro FGK-10x z-wave plus sensor tamper alarm not stopping

### ISSUE THREAD

**Issue by danone1973** [athombv/com.fibaro#168]
Homey 1.5.0-rc5
Fibaro app 1.5.7

Fibaro Doow/window sensor z-wave plus Australian frequency
Product Type ID 1793
Product ID 12289

After inclusion the tamper alarm will not stop. Sensor brand new and when included in Fibaro HC2 tamper alarm stops as is supposed to do.
Also the LED does not acknowledge the open/close contact although in Homey open/close works correctly.
The settings are attached.
<img width="613" alt="screen shot 2017-08-12 at 9 12 41 pm" src="https://user-images.githubusercontent.com/30952373/29240176-10f481b8-7fa3-11e7-8e44-4c3265f52db1.png">

**Comment by RobinBol** [comment]
Have you tried forcing another tamper alarm after pairing and waiting for it to turn of? Might be that due to the heavy communication during pairing the 'off' message was lost.

Have you tried changing the LED indication setting and waking up the device?

**Comment by danone1973** [comment]
Hello RobinBol, yes, I had tried all the above suggestions just before I created this issue. I have excluded/included numerous times the sensor as I tougth the same that maybe due some heavy traffic some messages got lost. No success.

However, just after the latest Fibaro app update (some 24 hours ago) the alarm tamper has been working correctly. I have monitored the sensor and I can say that no problems on the past 24 hours.
Thank you for sorting this out!
